src/problem-analysis/presentation.tex

```diff
@@ -0,0 +1,27 @@
+% textidote: ignore begin
+\section{Presentation of the problem}\label{sec:presentation-of-the-problem}
+% textidote: ignore end
+
+Several shortcomings from NOVA have been documented in the transcript in Figure~\ref{fig:problem-transcript}.
+The most noticeable is the issues regarding the~\acrshort{epos} insights and how they are not elaborate enough in its
+current state.
+
+\begin{figure}[h]
+    \centering
+    \large\textbf{“}
```

Comment:
@MadsHeilmann You want to use something like `\blockcquote` from `csquotes` (https://mirrors.dotsrc.org/ctan/macros/latex/contrib/csquotes/csquotes.pdf) or check https://www.overleaf.com/learn/latex/Typesetting_quotations